gyp/LICENSE

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2009 Google Inc. All rights reserved.
+Copyright (c) 2019 Refael Ackermann<refack@gmail.com>. All rights reserved.
```

Comment:
we might need to get legal confirmation that a stacked copyright is possible, my understanding is that it's locked in to the original claimant and you just have to live with it.

Reply:
Stacked copyright is the legally recommended way to claim rights of forked work. In this case, Google maintains the rights to work done till the fork.